community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/CypherCompiler.scala

```diff
@@ -74,23 +75,34 @@ object CypherCompilerFactory {
 
     val pickedPlannerName = plannerName.getOrElse(FallbackPlannerName)
```

Comment:
I still find it confusing that we define defaults in `RuntimeName` and `PlannerName` but we don't use these defaults here?